roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -11,6 +11,21 @@
   set_fact:
     is_ostree: "{{ ostree.stat.exists }}"
 
+- name: get ostree version
+  shell: "set -o pipefail && rpm-ostree --version |grep 'Version:' |awk '{print $2}'| sed \"s/'//g\" "
+  args:
+    executable: /bin/bash
+  register: ostree_version_on_server
+  when: is_ostree
+
+- name: set ostree version
+  set_fact:
+    ostree_version: "{{ ostree_version_on_server.stdout }}"
+  when:
+    - is_ostree
+    - 'ostree_version_on_server.stdout is defined'
```

Comment:
The code would be cleaner if you just checked `ostree_version_on_server is succceded` (also not a fan of this extremely descriptive name) in case rpm-ostree is not installed and allowed the previous task to ignore the failure.

Reply:
Yeah, it's smells not good, thanks again.